SPECS/keda/keda.spec

```diff
@@ -65,6 +68,12 @@ cp ./bin/keda-adapter %{buildroot}%{_bindir}
 %{_bindir}/%{name}-adapter
 
 %changelog
+* Mon Feb 05 2024 Nicolas Guibourge <nicolasg@microsoft.com> - 2.4.0-18
+- Patch CVE-2021-44716
+
+* Mon Feb 05 2024 Daniel McIlvaney <damcilva@microsoft.com> - 2.4.0-17
```

Comment:
This is just a missed changelog entry from the PR fixing CVE-2023-44487. All functional changes related to this entry are already inside the spec.